### PR TITLE
Always use LE for ppc64le.

### PR DESCRIPTION
**Pull request by yongtang**
### 1. What does this pull request do?

Use ppc64le.

### 2. Which issues (if any) are related?

This PR is sealed to #1177

### 3. Which documentation changes (if any) need to be made?
N/A

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by miekg** [comment]
I've pushed this change straight to master :)

**Comment by miekg** [comment]
I rather base it on LINUX_ARCHS

**Comment by yongtang** [comment]
@miekg I updated the PR so that now we only have one place `PLATFORMS` to obtain the list of platforms (for manifest-tool) and the list of `arch`s. Please take a look.

Also I think I might have push a image incorrectly to DockerHub and override the previous images inadvertently.

Can you push the Docker images again?

**Comment by miekg** [comment]
docker repushed

File: Makefile.release [hunk 1]
```diff
@@ -51,7 +51,8 @@ GITHUB:=coredns
 DOCKER:=coredns
 DOCKER_IMAGE_NAME:=$(DOCKER)/$(NAME)
 GITCOMMIT:=$(shell git describe --dirty --always)
-LINUX_ARCH=amd64 arm arm64 ppc64le s390x
+PLATFORMS:=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
+LINUX_ARCH:=$(shell echo $(PLATFORMS) | sed -e "s/linux\///g" | tr "," " ")
```
Comment (miekg) [inline Makefile.release]: isn't there a gnu make builtin for this? Either to create the PLATFORMS list or vice versa?

**Comment by yongtang** [comment]
@miekg Update with builtin `subst`. Please take a look.

**Pull request merged**